pkg/kubelet/stats/cri_stats_provider_linux.go

```diff
@@ -0,0 +1,65 @@
+//go:build linux
```

Comment:
@jsturtevant does it make sense to move this to cri_stats_providers_other.go instead? I am not quite sure so looking for feedback here.

Reply:
given that sandbox stats isn't implemented for `other` in containerd runtime (not sure about other runtimes) i think it makes sense to have `_linux` and `_other` for now?

@haircommander @bobbypage thoughts?